coverage/coverage_test.go

```diff
@@ -1518,29 +1495,37 @@ func testCoverage(t *testing.T, tc testCase) (*coverage.Model, error) {
 		return nil, fmt.Errorf("expand model: %+v", err)
 	}
 
-	//out, err := json.MarshalIndent(model, "", "\t")
-	//if err != nil {
-	//	t.Error(err)
-	//}
-	//t.Logf("expand model %s", string(out))
+	// out, err := json.MarshalIndent(model, "", "\t")
+	// if err != nil {
+	// 	t.Error(err)
+	// }
+	// t.Logf("expand model %s", string(out))
 
 	for _, rq := range tc.rawRequest {
 		request := map[string]interface{}{}
 		err = json.Unmarshal([]byte(rq), &request)
 		if err != nil {
-			t.Error(err)
+			t.Errorf("error unmarshal request json: %v", err)
 		}
 
 		model.MarkCovered(request)
 	}
 
 	model.CountCoverage()
 
-	//out, err = json.MarshalIndent(model, "", "\t")
-	//if err != nil {
-	//	t.Error(err)
-	//}
-	//t.Logf("coverage model %s", string(out))
+	if model.RootCoveredCount != tc.expectedCoveredCount {
+		t.Errorf("expected CoveredCount %d, got %d", tc.expectedCoveredCount, model.RootCoveredCount)
+	}
+
+	if model.RootTotalCount != tc.expectedTotalCount {
+		t.Errorf("expected TotalCount %d, got %d", tc.expectedTotalCount, model.RootTotalCount)
+	}
```

Comment:
It's checked in here for every case, and to cope with root level discriminator, `RootTotalCount` and `RootCoveredCount` are added to track the properties count, becase properties count variant of the  root model should NOT be added to the total count of root model